rpcs3/Emu/Cell/Modules/sceNp.cpp

```diff
@@ -392,7 +392,7 @@ error_code sceNpInit(u32 poolsize, vm::ptr<void> poolptr)
 	{
 		return SCE_NP_ERROR_INVALID_ARGUMENT;
 	}
-	else if (poolsize < SCE_NP_MIN_POOLSIZE)
+	else if ((s32)poolsize < SCE_NP_MIN_POOLSIZE)
```

Comment:
this makes no sense... just use u64 for poolsize in sceNpInit

Reply:
Fixed.